Node/Dockerfile

```diff
@@ -2,6 +2,7 @@ FROM ethereum/client-go
 
 RUN apk add --update git bash python3 musl-dev gcc python3-dev py3-netifaces
 RUN pip3 install web3
+RUN pip3 install websocket-client
```

Comment:
You can do this in one pip3 call, so only one pip command and one temp docker-image needs to be created:
`RUN pip3 install web3 websocket-client`